library/std/src/sys/pal/windows/pipe.rs

```diff
@@ -207,17 +206,6 @@ pub fn spawn_pipe_relay(
     Ok(theirs)
 }
 
-fn random_number() -> usize {
-    static N: AtomicUsize = AtomicUsize::new(0);
-    loop {
-        if N.load(Relaxed) != 0 {
-            return N.fetch_add(1, Relaxed);
-        }
-
-        N.store(hashmap_random_keys().0 as usize, Relaxed);
-    }
-}
```

Comment:
Removing this entirely changes the implementation. I'd prefer to just replace `hashmap_random_keys` here and leave any consequential changes to another PR.